shadow-dom/focus-navigation/focus-navigation-web-component-input-type-radio.html

```diff
@@ -8,7 +8,7 @@
 <script src="resources/focus-utils.js"></script>
 
 <template id="custom-radio">
-  <input type="radio">
+  <span aria-role="radio" tabindex="0">&#x1F518;</span>
```

Comment:
The test name should be changed, `input-type-radio` is no longer accurate.

Though I wonder if there's any point to this test now that we're changing it to a span? Will leave it to your judgement :)

Reply:
Oh, that's a good point. I'd rename it to focus-navigation-web-component-radio.html